lib/foreman/util.rb

```diff
@@ -22,9 +22,10 @@ def secure_token
       SecureRandom.base64(96).tr('+/=', '-_*')
     end
 
-    # recommended to make encryption_key at least 32 bytes. Ex. SecureRandom.hex(20)
+    # recommended to make encryption_key 32 bytes, matching the key length preferred by
+    # AS::MessageEncryptor's default algorithm
     def secure_encryption_key
-      SecureRandom.hex(20)
+      SecureRandom.hex(ActiveSupport::MessageEncryptor.key_len / 2)
```

Comment:
Nitpic: while touching this, could you replace hex with base64 or anything that is higher entropy? I understand that hex is easy to handle (just divide by two), but why no to be more secure by default? Base64 is still printable, the only drawback is slash char but we do not store this in any kind of filename.